torch/quantization/make_module.py

```diff
@@ -0,0 +1,50 @@
+import torch
+
+_operation_from_float = {
+    torch.add: torch.ops.quantized.add,
+    torch.cat: torch.ops.quantized.cat
+}
+
+"""Factory method to create a module wrapper.
+
+Args:
+    op: The operation to wrap the module around
+    quantized: Flag to indicate that the quantized counterpart is required.
+
+Returns:
+    Module class
+
+Example::
+
+    >>> add = make_module(torch.add)
+    >>> q_add = make_module(torch.add, quantized=True)
+"""
+def make_module(op=None, quantized=False):
+    name = op.__name__.capitalize()
+    name = "_ops_" + name
+
+    if quantized:
+        name = name = name + "_quantized"
+        op = _operation_from_float.get(op, op)
```

Comment:
you need to produce a nicer error message if the op is not found

Reply:
So is this? Because we won't be able to generate the files if there are no ops.